src/channel_server/drops_packet.cpp

```diff
@@ -72,23 +68,31 @@ PACKET_IMPL(dont_take) {
 	return builder;
 }
 
-PACKET_IMPL(remove_drop, game_map_object drop_id) {
+PACKET_IMPL(remove_drop, game_map_object drop_id, drop_despawn_types type, int32_t option) {
```

Comment:
This API could be made clearer; instead of passing what amounts to an overloaded int32_t, I would recommend:

- Separate functions that better describe the purpose
- A `struct` argument that has more descriptive members filled depending on which type of packet it is
- Having `optional<>` arguments with the different optional field information

I don't currently have an opinion on which is the best.